dotnet/xml/ns-Microsoft.ML.Transforms.TimeSeries.xml

```diff
@@ -1,7 +1,7 @@
 <Namespace Name="Microsoft.ML.Transforms.TimeSeries">
   <Docs>
-    <summary>A set of time series transfomer, estimator and their utilities, 
-      used for <a href="https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/tasks#anomaly-detection"> anomaly detection.</a>
+    <summary>
+        Namespace containing time-series data transformation components used for <a href="https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/tasks#anomaly-detection">anomaly detection.</a>
```

Comment:
> used for <a href="https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/tasks#anomaly-detection">anomaly detection.</a> [](start = 71, length = 133)

let's remove this. I think they can also do just time series analysis with them.

Reply:
ok